patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -68,7 +68,7 @@
          CreativeTabs creativetabs = this.func_77640_w();
          return creativetabs != null && (p_194125_1_ == CreativeTabs.field_78027_g || p_194125_1_ == creativetabs);
      }
-@@ -435,11 +445,704 @@
+@@ -435,11 +441,704 @@
```

Comment:
This file should not be changed, please regenerate your patches and leave out any file that you did not edit.